tests/dns/dns-rcode/test.yaml

```diff
@@ -0,0 +1,18 @@
+requires:
+  min-version: 8
+
+pcap: ../../dns-eve-v2-udp-nxdomain-soa/dns-udp-nxdomain-soa.pcap
+
+checks:
+  - filter:
+      count: 1
+      match:
+        alert.signature_id: 1
+        direction: to_client
+        app_proto: dns
+        event_type: alert
+        dns.answer.rcode: NXDOMAIN
+  - filter:
+      count: 1
+      match:
+        alert.signature_id: 2
```

Comment:
The rules are a bit confusing. Could we also have the `dns` and `flow` events here corresponding to these alerts so we see that the matches are happening correctly?